requirements/test-requirements.in

```diff
@@ -0,0 +1,12 @@
+-r requirements.txt
+
+beautifulsoup4>=4.4.1
+coverage
+-e git+git://github.com/millerdev/django-nose@fast-first-1.4.4#egg=django-nose
+fakecouch==0.0.10
```

Comment:
How'd we end up pinning to this number? The original (pre-PR) seems to be `fakecouch==0.0.15`? Test failure seems be related to this (only in that it mentions `fakecouch`)

Reply:
I'm not sure what's going on with this file! I just ran `make requirements` and that's it

Reply:
Changed it back to 0.0.15 without a problem so far:
https://github.com/dimagi/commcare-hq/pull/21411/commits/b542b3c6830be7cc2dcdc53c79143e088eb5af7c

Any idea how this could have happened? If 15 was necessary, this should have caught it, but maybe some dependency itself isn't properly pinned?

Reply:
@pr33thi this is the .in file, so (hopefully) you wrote this file

Reply:
But it looks like this was something that was updated on master after you started this PR, so that explains it.

Reply:
Looks like this commit (b542b3c) didn't end up in this PR